pkgdb/src/flox-flake.cc

```diff
@@ -67,7 +67,7 @@ callInChildProcess( std::function<void()>  lambda,
             }
           /* The error has already been reported via the child, just pass
            * along the exit code. */
-          exit( WEXITSTATUS( status ) );
+          _exit( WEXITSTATUS( status ) );
```

Comment:
question: what is the practical difference between `_exit` and `exit` here?
Is that expected to be known or does it warrant a comment/breadcrumb?

Reply:
I'll comment it. `_exit()`  does not run the `atexit` handlers I think.